http/src/v0/block.rs

```diff
@@ -46,54 +47,32 @@ pub struct PutQuery {
     version: Option<u8>,
 }
 
-#[derive(Debug, Serialize)]
-#[serde(rename_all = "PascalCase")]
-pub struct PutResponse {
```

Comment:
Deleted `PutResponse` as the `serde_json::json!` is better suited for such moving things (cannot benefit from any `Cow<'_, str>`) as discovered originally http/src/v0/dag.rs.